plugins/modules/aci_l3out.py

```diff
@@ -411,8 +411,8 @@ def main():
             module.fail_json(msg="Parameter 'asn' is required when l3protocol is 'eigrp'")
         if "eigrp" not in l3protocol and asn is not None:
             module.warn("Parameter 'asn' is only applicable when l3protocol is 'eigrp'. The ASN will be ignored")
-        if "ospf" in l3protocol and ospf is None:
-            module.fail_json(msg="ospf specifications are required when l3protocol is 'ospf'")
+        # if "ospf" in l3protocol and ospf is None:
+        #     module.fail_json(msg="ospf specifications are required when l3protocol is 'ospf'")
```

Comment:
Can be removed if not needed. Did you test the scenario of providing "ospf" in l3protocol without specifying ospf?

Reply:
yes, I did and it's working fine. It's providing the required attributes with their default values.